pkg/kubelet/cm/container_manager_stub.go

```diff
@@ -43,12 +43,14 @@ import (
 type containerManagerStub struct {
 	shouldResetExtendedResourceCapacity bool
 	extendedPluginResources             v1.ResourceList
+	cpuManager                          cpumanager.Manager
 }
 
 var _ ContainerManager = &containerManagerStub{}
 
-func (cm *containerManagerStub) Start(_ context.Context, _ *v1.Node, _ ActivePodsFunc, _ GetNodeFunc, _ config.SourcesReady, _ status.PodStatusProvider, _ internalapi.RuntimeService, _ bool) error {
+func (cm *containerManagerStub) Start(ctx context.Context, _ *v1.Node, _ ActivePodsFunc, _ GetNodeFunc, _ config.SourcesReady, _ status.PodStatusProvider, _ internalapi.RuntimeService, _ bool) error {
 	klog.V(2).InfoS("Starting stub container manager")
+	cm.cpuManager = cpumanager.NewFakeManager(ctx)
```

Comment:
Are we intentionally leaving  `klog.InfoS` and not updating to the logger equivalent in the line above? If so, why?

Reply:
Yes, same reasoning per https://github.com/kubernetes/kubernetes/pull/125912#discussion_r1950480214 . I'm trying to do the minimal changes and move gradually once the deps are settled. This will cause some churn, but it's the only way I can think of to keep PRs size manageable.